server/gfmd/db_ldap.c

```diff
@@ -3264,6 +3264,162 @@ gfarm_ldap_quota_dir_load(void *closure,
 	return (GFARM_ERR_FUNCTION_NOT_IMPLEMENTED);
 }
 
+/**********************************************************************/
+
+static char *gfarm_ldap_user_auth_make_dn(void *vkey);
+static gfarm_error_t gfarm_ldap_user_auth_set_field(void *info,
+	char *attribute, char **vals);
+
+struct gfarm_ldap_user_auth_key {
+	const char *username;
+	const char *auth_method;
+};
+
+static const struct gfarm_ldap_generic_info_ops gfarm_ldap_user_auth_ops = {
+	&gfarm_base_user_auth_ops,
+	"(objectclass=GFarmUserAuth)",
+	"username=%s, auth_method=%s, %s",
+	gfarm_ldap_user_info_make_dn,
+	gfarm_ldap_user_info_set_field,
+};
+
+static char *
+gfarm_ldap_user_auth_make_dn(void *vkey)
+{
+	struct gfarm_ldap_user_auth_key *key = vkey;
+	char *dn;
+
+	GFARM_MALLOC_ARRAY(dn, strlen(gfarm_ldap_user_auth_ops.dn_template) +
+		strlen(key->username) + strlen(key->auth_method) +
+		strlen(gfarm_ldap_base_dn) + 1);
+	if (dn == NULL) {
+		gflog_debug(GFARM_MSG_1002128,
```

Comment:
change GFARM_MSG_1002128 to GFARM_MSG_UNFIXED

Reply:
commit [429eb3d](https://github.com/oss-tsukuba/gfarm/pull/1141/commits/429eb3daa151165a09117234d565149a2a1ac4e6) has not changed GFARM_MSG_1002128 to GFARM_MSG_UNFIXED yet.
Could you change it?